lib/rspec/core/shell_escape.rb

```diff
@@ -0,0 +1,49 @@
+module RSpec
+  module Core
+    # @private
+    # Deals with the fact that `shellwords` only works on POSIX systems.
+    module ShellEscape
```

Comment:
I think it could make sense to move it there if any of the rspec libs need this, but that hasn't happened and I'd be surprised if it did -- dealing with shell escaping is pretty rspec-core specific since it's what deals with loading files off disk, etc.